service/src/test/java/com/kodemon/service/test/TimeServiceTest.java

```diff
@@ -29,10 +29,10 @@
 
     @Test
     public void currentDateTest() {
+        long allowedReserve = 5L;
```

Comment:
I said static final constant and this is non of that :smile:. But merge it :smile: